### PR TITLE
add sources from clickforce holmes ads

### PR DESCRIPTION
**Pull request by MIGOdanis**

### CODE REVIEW

**Comment by MIGOdanis** [comment]
@googlebot I signed it!

File: organizations/clickforce.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "Clickforce",
```
Comment (pfrisella) [inline organizations/clickforce.json:2]: CLICKFORCE

File: sources/holmesads.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "HOLMES ADS",
```
Comment (pfrisella) [inline sources/holmesads.json:2]: it should match the filename.
HOLMESADS

File: sources/holmesads.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "HOLMES ADS",
+  "categories": ["ADVERTISING"],
+  "organization": "Clickforce",
```
Comment (pfrisella) [inline sources/holmesads.json:5]: CLICKFORCE

**Review by pfrisella — CHANGES_REQUESTED** [review]

**Comment by MIGOdanis** [comment]
@pfrisella thank you.  I fixed it.

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for adding!

**Pull request merged**